app/src/main/java/com/example/habitsmasher/ui/profile/FollowListFragment.java

```diff
@@ -60,24 +62,22 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
 
         getBundleArguments();
         // Set header title
-        Objects.requireNonNull(((AppCompatActivity) requireActivity()).getSupportActionBar()).setTitle(_followType + " List");
+        Objects.requireNonNull(((AppCompatActivity) requireActivity()).getSupportActionBar()).setTitle(_followType);
 
         // query firebase for all habits that correspond to the current user
         Query query = getListFromFirebase();
 
-        if(query != null){
-            // populate the list with existing items in the database
-            FirestoreRecyclerOptions<User> options = new FirestoreRecyclerOptions.Builder<User>()
-                    .setQuery(query, User.class)
-                    .build();
-
-            setFollowList();
-            // set the item adapter
-            _followItemAdapter = new FollowItemAdapter(options, _followList, _user.getUsername());
-            // Inflate follow list fragment
-            LinearLayoutManager layoutManager = new LinearLayoutManager(_context, LinearLayoutManager.VERTICAL, false);
-            initializeRecyclerView(layoutManager, view);
-        }
+        // populate the list with existing items in the database
+        FirestoreRecyclerOptions<User> options = new FirestoreRecyclerOptions.Builder<User>()
+                .setQuery(query, User.class)
+                .build();
+
+        setFollowList();
+        // set the item adapter
+        _followItemAdapter = new FollowItemAdapter(options, _followList, _user.getUsername());
```

Comment:
_user.getUsername() should be _user.getId(), but it's not massive since we never end up using the field anyways